ChangeLog.md

```diff
@@ -1,6 +1,11 @@
 Note: This is an Azure Storage only package. The all up Azure node sdk still has the old storage bits in there. In a future release, those storage bits will be removed and an npm dependency to this storage node sdk will 
 be taken. This is a GA release and the changes described below indicate the changes from the Azure node SDK 0.9.8 available here - https://github.com/Azure/azure-sdk-for-node.
 
+2016.10 Version 1.3.2
+
+BLOB
+* Prevent a blockId from being generated with a decimal point.
+
```

Comment:
It's not a breaking change. 

1. TSC will help check the error and give the detailed information. 
2. "any" support everything UUID supports. :)